pkgs/applications/version-management/git-and-tools/git-subrepo/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchFromGitHub, git, which }:
+
+stdenv.mkDerivation rec {
+  name = "git-subrepo-${version}";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ingydotnet";
+    repo = "git-subrepo";
+    rev = "${version}";
+    sha256 = "05m2dm9gq2nggwnxxdyq2kjj584sn2lxk66pr1qhjxnk81awj9l7";
+  };
+
+  buildInputs = [
+    git which
```

Comment:
Is `which` actually used in the program or for during the build?

Reply:
Yep, surprisingly! https://github.com/ingydotnet/git-subrepo/blob/5d6aba91dbff3157e498b0a795e99e2fcb7d9ec4/Makefile#L2

Reply:
Ah I see. Can you move `which` to `nativeBuildInputs` then?

Also, I guessing this hard depends on the git executables, can you wrap the program like?
```
nativeBuildInputs = [
  makeWrapper
];

postFixup = ''
  wrapProgram $out/bin/$program_name \
    --prefix PATH : "${git}/bin"
'';